plugin-server/src/cdp/services/fetch-executor.service.ts

```diff
@@ -29,6 +31,97 @@ export const RETRIABLE_STATUS_CODES = [
     504, // Gateway Timeout
 ]
 
+const parseLiquidTemplate = (
```

Comment:
This is 100% the wrong place for this but good attempt. Will PR with the right place